packages/engine/src/scene/components/PostProcessingComponent.tsx

```diff
@@ -45,12 +44,8 @@ export const PostProcessingComponent = defineComponent({
   onSet: (entity, component, json) => {
     if (!json) return
 
-    if (json.enabled) component.enabled.set(json.enabled)
-    if (json.effects) {
-      for (const [name, effect] of Object.entries(json.effects)) {
-        component.effects[name].merge(effect)
-      }
-    }
+    if (typeof json.enabled === 'boolean') component.enabled.set(json.enabled)
+    if (typeof json.effects === 'object') component.effects.set(json.effects)
```

Comment:
we need to deeply clone any objects to avoid mutating original scene data. the merge previously is also to avoid removing effects that are not part of the current saved schema